src/xrpld/rpc/handlers/LedgerEntry.cpp

```diff
@@ -837,43 +889,51 @@ doLedgerEntry(RPC::JsonContext& context)
             // For apiVersion 2 onwards, any parsing failures that throw
             // this
```

Comment:
nit: Looks like a formatting issue, maybe due to `clang-format` and column width limit changes in the past

Reply:
Fixed in https://github.com/XRPLF/rippled/pull/5199/commits/3d5ef0fb972cf519f9c890126e2364e5316a014e